### PR TITLE
Beta7

### PR DESCRIPTION
**Pull request by JimBobSquarePants**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
- [x] Updated submodules and replaced guarding code.
- [x] Updated project dependencies to latest nuget betas
- [x] Merged #984 
- [x] Merged #989 

I haven't enabled StyleCop for the tests yet as it's a days worth of work to cleanup all the issues there and it can wait until after the release. If someone can please review the linked PRs and see if they are in a mergeable state. The bug fix must definitely go in, the enhancement can possibly wait.

### CODE REVIEW

File: src/ImageSharp.Drawing/ImageSharp.Drawing.csproj [hunk 1]
```diff
@@ -11,6 +11,15 @@
     <TargetFrameworks>netcoreapp2.1;netstandard1.3;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
+  <!-- TODO: Include .NETSTANDARD2.1 when released-->
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp2')) ">
```
Comment (antonfirsov) [inline src/ImageSharp.Drawing/ImageSharp.Drawing.csproj]: 👍 cool stuff

**Comment by antonfirsov** [comment]
image

Ooops I think I did this by accident ...

**Comment by JimBobSquarePants** [comment]
@antonfirsov the minimum version for Core etc is specified already in the targets file. Let’s ship the equaliser as-is and raise a new ticket.

File: Directory.Build.targets [hunk 2]
```diff
@@ -28,10 +28,10 @@
     <PackageReference Update="Microsoft.Net.Compilers.Toolset" Version="3.1.0" />
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Update="Moq" Version="4.10.0" />
-    <PackageReference Update="SixLabors.Core" Version="1.0.0-dev000101" />
-    <PackageReference Update="SixLabors.Fonts" Version="1.0.0-beta0008" />
-    <PackageReference Update="SixLabors.Shapes" Version="1.0.0-beta0008" />
-    <PackageReference Update="SixLabors.Shapes.Text" Version="1.0.0-beta0007" />
+    <PackageReference Update="SixLabors.Core" Version="1.0.0-beta0008" />
```
Comment (JimBobSquarePants) [inline Directory.Build.targets]: @antonfirsov see here for min versions

Comment (antonfirsov) [reply Directory.Build.targets]: Thanks, totally missed this.

**Comment by Rwing** [comment]
Is there release schedule? I can't wait to use the new version

**Comment by JimBobSquarePants** [comment]
@Rwing Most likely within the nest 12 hours.

**Pull request merged**